app/test/logic/Contacts/VCard/TestData/Thunderbird.json

```diff
@@ -28,7 +28,12 @@
     "preference": 100
   }],
   "streetAddresses": [{
-    "value": "My Street\nMy City\n00000\nMy State\nGermany",
+    "instructions": null,
+    "street": "My Street",
+    "city": "My City",
+    "state": "My State",
+    "postCode": "00000",
```

Comment:
`postalCode` (with your coming fix)

Reply:
So which PR do I put the `postalCode` fix in? Probably this one, actually.